_posts/2018-10-17-Tiskova-zprava-k-povolebnimu-vyjednavani

```diff
@@ -1,10 +1,10 @@
 ---
 title:    "Tisková zpráva k aktuální situaci povolebního vyjednávání"
-layout:	  post
-category: blog
-author:	  Kateřina Stojanová
+layout:	  "post"
+category: "blog"
```

Comment:
tady je to podle mě zbytečné (jednoslovný řetězec bez diakritiky, ale nemělo by to nic zkazit, IMO)